src/core/lib/channel/channelz_registry.cc

```diff
@@ -58,19 +45,24 @@ ChannelzRegistry* ChannelzRegistry::Default() {
   return g_channelz_registry;
 }
 
-ChannelzRegistry::ChannelzRegistry() : uuid_(1) {
+ChannelzRegistry::ChannelzRegistry() {
   gpr_mu_init(&mu_);
-  avl_ = grpc_avl_create(&avl_vtable);
+  // init elements to nullptr
```

Comment:
I don't think there's any need for this, since `InlinedVector` tracks its own size.

Reply:
Done